inabox/templates/testconfig-anvil.yaml

```diff
@@ -36,6 +36,16 @@ services:
     operators: 4
     maxOperatorCount: 3
     relays: 4
+  proxy:
+    enabled: true
+    rest_port: 3100
+    arb_port: 3101
+    max_blob_size: "1mib"
+    enabled_apis:
+      standard_commitment: true
+      op_generic_commitment: true
+      op_keccak_commitment: true
+    enable_arb_custom_da: true
```

Comment:
@dmanc wdyt? does this fit with what you had in mind? other services are started with ENV_VARS below. should we also use that, or...? whats the long-term plan?

Reply:
This seems like how I envisioned configuring each component. Long-term plan is to deprecate the env var based deployment below.